### PR TITLE
Fix chart not showing any data for single entry

### PR DESCRIPTION
**Pull request by Anshuman71**
Change the assignment of  0 with 1 when we add it for the 1st time to object #119 #120

### CODE REVIEW

File: dashboard/controller/stats.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ function getHowUserLaunchActivity(req, res) {
 			var d = {}
 			for (var i = 0; i < data.length; i++) {
 				if (!d.hasOwnProperty(data[i].event_object)) {
-					d[data[i].event_object] = 0
+					d[data[i].event_object] = 1
```
Comment (NikhilM98) [inline dashboard/controller/stats.js]: If you set it to 1, then it'll generate the chart, but it'll show incorrect data. It'll show some has already launched one activity from the `List View` and one from the `Journal View`.

Comment (ahmedkrmn) [reply dashboard/controller/stats.js]: I've doubled checked and it's working correctly for that case. Can you please elaborate, maybe some screenshots?

**Comment by NikhilM98** [comment]
> If you set it to 1, then it'll generate the chart, but it'll show incorrect data.
> When you hover on the chart, it'll show that some user has already launched one activity from the `List View` and one from the `Journal View`.

@Anshuman71 @ahmedkrmn I checked the data again. I admit my last analysis was incorrect.
This patch will not only solve the issue with the rendering of the chart for single data, but it'll also fix the count which was initially started from the wrong number.

**Comment by Anshuman71** [comment]
> > If you set it to 1, then it'll generate the chart, but it'll show incorrect data.
> > When you hover on the chart, it'll show that some user has already launched one activity from the `List View` and one from the `Journal View`.
> 
> @Anshuman71 @ahmedkrmn I checked the data again. I admit my last analysis was incorrect.
> This patch will not only solve the issue with the rendering of the chart for single data, but it'll also fix the count which was initially started from the wrong number.

Thank you @NikhilM98 for the feedback and support.

**Comment by Anshuman71** [comment]
@tarunsinghal92  please review this change.

**Comment by llaske** [comment]
@Anshuman71 you're right. The count was wrong. Thanks.

**Pull request merged**